src/component/common/Table/TableContainer/TableContainer.tsx

```diff
@@ -0,0 +1,15 @@
+import { forwardRef } from 'react';
+import { Paper, PaperProps } from '@mui/material';
+import { useStyles } from './TableContainer.styles';
+
+export const TableContainer = forwardRef<HTMLDivElement, PaperProps>(
+    ({ children, ...props }, ref) => {
+        const { classes: styles } = useStyles();
```

Comment:
The `classes: styles` destructuring was done to make the MUI migration diff smaller. 

Can also use `classes` directly.